initfiles/componentfiles/configxml/esp.xsd.in

```diff
@@ -863,10 +863,17 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="memCachedOptionString" type="xs:string" use="optional">
+            <xs:attribute name="ensureESPCache" type="xs:boolean" use="optional" default="true">
```

Comment:
If an existing ESP is upgraded, doesn't have added config, and doesn't have memcached running, it will continue to work?

Also have you asked others what they think should happen for NEW ESPs if memcached is not installed.

I'm starting to think maybe we should default to OFF, but if its ON fail to start if we can't find memcached.  we definitely want to encourage it's use and not accidentally have systems running without caching that are meant to have it.  But should a simple installation fail by default without it?